wavedump/src/dark_vs_laser.py

```diff
@@ -0,0 +1,73 @@
+import csv
```

Comment:
Can you add a comment string above explaining what this file is used for, something like:

```
"""
Script used to plot the average SPE charge using the dark counts and the laser...

Author: Kai Svenson
Date: August 4, 2022
"""